rust/agama-server/tests/common/mod.rs

```diff
@@ -41,6 +41,12 @@ pub trait ServerState {}
 impl ServerState for Started {}
 impl ServerState for Stopped {}
 
+impl Default for DBusServer<Stopped> {
+    fn default() -> Self {
```

Comment:
This was automatically added by the clippy check, no idea why... :thinking:

Reply:
https://rust-lang.github.io/rust-clippy/v0.0.212/index.html#new_without_default